src/Core/Kernel.php

```diff
@@ -61,26 +64,27 @@ public function __construct(
         private string $cacheId,
         string $version,
         Connection $connection,
-        protected string $projectDir
+        protected string $projectDir,
+        ?FilesystemOperator $filesystem = null,
```

Comment:
why using league flysystem? We don't need any filesystem abstraction. any reason against symfony filesystem class for this purpose?

Reply:
as discussed in Slack, I will change to Symfony Filesystem :+1: 

there is not need for the League Flyssystem, and we should keep the kernel as simple as possible